src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1046,7 +1050,8 @@ void OrbitMainWindow::on_actionToggle_Capture_triggered() { app_->ToggleCapture(
 
 const QString OrbitMainWindow::kEnableCallstackSamplingSettingKey{"EnableCallstackSampling"};
 const QString OrbitMainWindow::kCallstackSamplingPeriodMsSettingKey{"CallstackSamplingPeriodMs"};
-const QString OrbitMainWindow::kCallstackUnwindingMethodSettingKey{"CallstackUnwindingMethod"};
+// Deliberatly changed to "CallstackUnwindingMethod2" since encoding in the settings was changed.
+const QString OrbitMainWindow::kCallstackUnwindingMethodSettingKey{"CallstackUnwindingMethod2"};
```

Comment:
The alternative to this slightly ugly hack would be to sanitize the reading from the settings to make sure it returns one of the valid values. This would need to happen in `LoadCaptureOptionsIntoApp `and `on_actionCaptureOptions_triggered` and look roughly like this:
```
if (method != kDwarf && method != kFramePointers) {
  method = kCallstackUnwindingMethodDefaultValue;
}
```
Let me know if yoiu have a better idea...

Reply:
I think you can really just override it. That option was there for less then a week, only in devmode (never released), so in fact it will hit me, and maybe @dpallotti. And it might result in strange behaviour only the first capture run.

Reply:
I would be for keeping the old key (as Florian said, it doesn't matter for now) and defaulting to DWARF in case of a wrong input, which is something we probably want anyway.

Reply:
Reverted the name of the setting and added the sanatization to the places where the setting is read.